src/plotting/plot.component.tsx

```diff
@@ -82,9 +82,10 @@ const ConnectedPlot = (props: PlotProps) => {
   const chartData: ChartData<'scatter'> = React.useMemo(() => {
     const data =
       records?.map((record) => ({
-        x: parseInt(record.metadata.timestamp),
-        y: record.metadata.shotNum ?? NaN,
+        x: Math.floor(new Date(record.metadata.timestamp).getTime()),
```

Comment:
Ah this is leftover from when I was accidentally getting the number of milliseconds since epoch rather than seconds. It's not necessary so I'll change

Reply:
`getTime` does return milliseconds since epoch - it just returns it as in `number` though instead of a `float`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime